### PR TITLE
[MET-115] PHP 8.1 Support

### PR DESCRIPTION
**Pull request by DenisYaschuk**
[MET-114] Symfony 5 upgrade

### CODE REVIEW

File: src/Migrations/Cli/Command/GenerateMigrationCommand.php [hunk 1]
```diff
@@ -63,5 +63,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->migrationGenerator->generate($timestamp, $config['migrations']['namespace'], $path);
 
         $this->io->success(sprintf('Generated migration in %s/Version%s.php', $path, $timestamp));
+
+        return 0;
```
Comment (asmaa-saber) [inline src/Migrations/Cli/Command/GenerateMigrationCommand.php]: Can we please use the constant instead? \Symfony\Component\Console\Command\Command::SUCCESS

File: src/Migrations/Cli/Command/ExecuteMigrationCommand.php [hunk 2]
```diff
@@ -125,8 +125,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $direction
         );
 
-        if (!$result) {
-            return 1;
-        }
+        return $result ? 0 : 1;
```
Comment (asmaa-saber) [inline src/Migrations/Cli/Command/ExecuteMigrationCommand.php]: Can use the constants here too Command::SUCCESS and Command::FAILURE

File: src/Migrations/Cli/Command/GenerateMigrationCommand.php (hunk 1 above)
Comment (DenisYaschuk) [reply src/Migrations/Cli/Command/GenerateMigrationCommand.php]: Tested it and it didn't break anything so will replace it

File: src/Migrations/Cli/Command/ExecuteMigrationCommand.php (hunk 2 above)
Comment (DenisYaschuk) [reply src/Migrations/Cli/Command/ExecuteMigrationCommand.php]: There are a whole lot of places with similar stuff will go over everything I can find and try to refactor it

File: src/Migrations/Cli/Command/GenerateMigrationCommand.php (hunk 1 above)
Comment (asmaa-saber) [reply src/Migrations/Cli/Command/GenerateMigrationCommand.php]: Brill, thank you!
Would you please do so for all the Command classes? 
They're under
Migrations/Cli/Command/
Patch/Cli/Command/
Permissions/Cli/Command/
Scripts/Cli/Command

Comment (DenisYaschuk) [reply src/Migrations/Cli/Command/GenerateMigrationCommand.php]: Just pushed the changes, shall I mark this as resolved or will you take another look and resolve it yourself?

**Pull request merged**